src/Goerli-DssSpell.t.base.sol

```diff
@@ -947,17 +947,17 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
             calc_cut:     9900
         });
         afterSpell.collaterals["AAVE-A"] = CollateralValues({
-            aL_enabled:   true,
-            aL_line:      50 * MILLION,
-            aL_gap:       5 * MILLION,
-            aL_ttl:       8 hours,
+            aL_enabled:   false,
+            aL_line:      0,
+            aL_gap:       0,
+            aL_ttl:       0 hours,
```

Comment:
I'd remove the `hours` like in the other two.

Reply:
Done in 832c7e7.